modules/stb_vorbis/audio_stream_ogg_vorbis.h

```diff
@@ -52,18 +52,18 @@ class AudioStreamPlaybackOGGVorbis : public AudioStreamPlaybackResampled {
 	Ref<AudioStreamOGGVorbis> vorbis_stream;
 
 protected:
-	virtual void _mix_internal(AudioFrame *p_buffer, int p_frames) override;
-	virtual float get_stream_sampling_rate() override;
+	virtual void _mix_internal(AudioFrame *p_buffer, int p_frames);
+	virtual float get_stream_sampling_rate();
```

Comment:
Those are necessary in the `master` branch, but indeed if you did the work initially on `3.2` there might be some mismatch, as we weren't using the `override` keyword in `3.2`.